metadata-ingestion/src/datahub/ingestion/source/looker.py

```diff
@@ -557,38 +672,49 @@ def _get_looker_dashboard_element(  # noqa: C901
                         LookerExplore(model_name=element.look.query.model, name=exp)
                         for exp in explores
                     ],
-                    upstream_fields=fields,
+                    upstream_fields=upstream_fields,
```

Comment:
curious -> what is the diff between an upstream field and an input field?

Reply:
+1 we need to be very clear about this

Reply:
simplified things so we just track input fields now- input fields are a superset of upstream fields and also chart-specific fields